### PR TITLE
Improve/image scenes

### PR DESCRIPTION
**Pull request by pleszkiewicz**
Update props in multiple scenes.

### CODE REVIEW

File: src/test_scenes/sceneProgressBar.js [hunk 1]
```diff
@@ -38,15 +38,23 @@ class SceneProgressBar extends React.Component {
...
+        itemPadding={[
+          { index: 0, padding: [0, 0, 0.1, 0] },
+          { index: 4, padding: [0.15, 0, 0, 0] },
+        ]}
+        position={this.props.position}
+      >
         <ProgressBar 
-          alignment={'center-center'}
+          alignment={Alignment.centerCenter}
           position={[0, 0.2, 0]}
```
Comment (kpiascik) [inline src/test_scenes/sceneProgressBar.js]: do you still need the positions for all the progress bars?  You used them when it was a `View` but now `LinearLayout` doesn't need them.

File: src/test_scenes/sceneSlider.js [hunk 2]
```diff
@@ -22,45 +22,42 @@ class SceneSlider extends React.Component {
...
     const { defaultSliderValue, userDefinedSliderOneValue, userDefinedSliderTwoValue } = this.state;
     return (
-      <View position={this.props.position}>
-        <GridLayout
-          anchorPoint={"top-center"}
-          columns={1}
-          defaultItemPadding={[0.075, 0, 0, 0]}
-          position={[0.0, 1.0, 0.0]}
-        >
-          <Text alignment={"center-center"} fontSize={0.075} >
```
Comment (kpiascik) [inline src/test_scenes/sceneSlider.js:32]: you're no longer specifying `Alignement`?

File: src/test_scenes/sceneProgressBar.js (hunk 1 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneProgressBar.js]: right 🙇

File: src/test_scenes/sceneSlider.js (hunk 2 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneSlider.js:32]: 🙇

File: src/test_scenes/sceneProgressBar.js (hunk 1 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneProgressBar.js]: Fixed (removed values for `position` and `alignment` props).

**Review by kpiascik — APPROVED** [review]

**Pull request merged**